bolt_aix.go

```diff
@@ -8,6 +8,7 @@ import (
 	"time"
 	"unsafe"
 
+	"go.etcd.io/bbolt/internal/common"
```

Comment:
This import item should be put in a separate group. It seems that the bbolt workflow can't detect such minor issue.

cc @Elbehery @ivanvc